src/domain/accessibilityStatement/accessibilityStatement.module.scss

```diff
@@ -13,8 +13,4 @@
     word-break: break-word;
     hyphens: manual;
   }
-
-  p {
-    max-width: $textParagraphMaxWidth;
```

Comment:
Did you remove this on purpose?

Reply:
oh yes typo, will put it back, i tested it and commit by mistake